### DISCUSSION TITLE
REQ: hud_ping_pl_period, scr_scoreboard_ping_period, scr_scoreboard_pl_period

### DISCUSSION THREAD

**Discussion by Matrix-QW** [ezQuake/ezquake-source#513] [Ideas]
The scoreboard ping if i'm not mistaken is updated every 2 seconds.
With cl_earlypackets 1 it does not reflect real ping at all.
I use 0.5 for hud_ping and it's readable.
Period for pl would be useful.

Thank you.

**Comment by meag** [comment]
Not sure how to progress this: the scoreboard ping & packet loss is taken from the server's svc_updateping/svc_updatepl messages, which seem to be sent in response to the usercommand "pings".

Even if we reduced the delay, it still wouldn't show you a "real ping", as it would still be the server's point of view.

**Comment by Matrix-QW** [comment]
Actually the real ping is always what server allows, so it's 12.
I suppose the 1ms i see in localhost is just client app side. So forget about it.
Still, hud_ping_pl_period could be implemented.
Also, i notice the hud_ping is not always at pair with hud_net even with same period, so something is wrong.
What about implementing show ping/pl only if it reaches x ? Same way as hud_fps_drop works, but in this case values going up.
